skunkworks_crow/src/main/assets/open_source_licenses.html

```diff
@@ -28,6 +28,15 @@ <h3>
     Apache License Version 2.0, January 2004
 </h3>
 <ul>
+    <li>
+        <b>Permissions Dispatcher</b>
+    </li>
+    <li>
+        <b>Mockito</b>
```

Comment:
Mockito and Roboelectric are MIT licensed they should not come under Apache section.
You can refer https://github.com/opendatakit/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html for adding the MIT License section